planning/behavior_velocity_planner/src/scene_module/virtual_traffic_light/manager.cpp

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include "tier4_autoware_utils/math/unit_conversion.hpp"
+
 #include <scene_module/virtual_traffic_light/manager.hpp>
```

Comment:
Instead, this can be changed to `""`. (But no need to apply it in this PR. It's up to you.)

```suggestion
#include "scene_module/virtual_traffic_light/manager.hpp"
```

Reply:
If you fix this, please modify other files as well.

Reply:
I'll leave it as it is in this PR.